### PR TITLE
Cocoapods Integration

### PR DESCRIPTION
**Pull request by Strobocop**
Added a pod spec. Still need to push the spec to the cocoa pods spec repo, but I will leave that to you in case you decide you don't want to accept this PR.

### CODE REVIEW

File: Codability.podspec [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#
+#  Be sure to run `pod spec lint Codability.podspec' to ensure this is a
+#  valid spec and to remove all comments including this before submitting the spec.
+#
+#  To learn more about Podspec attributes see http://docs.cocoapods.org/specification.html
```
Comment (yonaskolb) [inline Codability.podspec]: Just need to remove these comments

File: Codability.podspec [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	s.name         = "Codability"
+	s.version      = "0.2.1"
+	s.swift_version = "4.1"
+	s.summary      = "Useful helpers for working with Codable types in Swift"
+	s.homepage     = "http://github.com/yonaskolb/Codability"
+	s.license      = { :type => "MIT", :file => "LICENSE" }
+	s.author       = { "Yonas Kolb" => "yonas4596@hotmail.com" }
+	s.social_media_url   = "https://twitter.com/yonaskolb"
+	s.ios.deployment_target = "9.0"
+	s.tvos.deployment_target = "9.0"
```
Comment (yonaskolb) [inline Codability.podspec]: We can add osx and watchOS as well

**Review by yonaskolb — COMMENTED** [review]
Fantastic, thank you @Strobocop!

**Comment by Strobocop** [comment]
Went ahead and added other platforms, but I frankly don't build too much on them so I am not wise enough to choose a good deployment_target. Nor was I sure how to test them easily without an example project setup. So I just used some deployment targets I have used on other libraries in the past. We may be able to go lower. Let me know if you care about the specific target and I'll change it.

**Comment by Strobocop** [comment]
@yonaskolb, I just wanted to check the status of this. If you happen to just be too busy, and pushing the spec to the cocoapods spec repo is a hinderance to the merging of this PR, I think it might still be worthwhile to merge. I believe the library could still be integrated with the git tag and no need to push to spec repo like so:

```swift
pod 'Codability', :git => 'http://github.com/yonaskolb/Codability'
```

**Comment by yonaskolb** [comment]
Sorry @Strobocop, I forgot about this!

**Comment by Strobocop** [comment]
No worries @yonaskolb. 

Still hoping to get this pushed to Cocoapods. From what I can tell, it is not yet. Just need to push the actual release to their spec repo. 

https://guides.cocoapods.org/making/getting-setup-with-trunk

**Comment by yonaskolb** [comment]
@Strobocop `0.2.1` pushed to trunk! 👍

**Pull request merged**